source/agora/consensus/data/Enrollment.d

```diff
@@ -16,6 +16,22 @@ module agora.consensus.data.Enrollment;
 import agora.common.Types;
 import agora.common.crypto.Schnorr : Signature;
 
+/*******************************************************************************
+
+    Define enrollment Set
+    This is an enrolment set with sort order values.
+
+*******************************************************************************/
+
+public struct EnrollmentSet
+{
+    /// the validators' key indices which enrollment validators
+    public ushort index;
```

Comment:
You could transfer this variable into the Enrollment structure and remove the EnrollmentSet structure. The name, EnrollmentSet is kind of confusing also which is not a matter if you remove it.

Reply:
applied the suggest

Reply:
Why do you need the index here ?

Reply:
I deleted index.